### PR TITLE
Feature/pi 1434 carthage testing

### PR DESCRIPTION
**Pull request by deepesh-vasthimal-cko**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [ *] Bugfix (non-breaking change which fixes an issue)
* [ *] New feature (non-breaking change which adds functionality)
* [* ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [ *] Lint and unit tests pass locally with my changes
* [ *] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by harry-brown-cko** [comment]
bitrise is failing due to ssh issues (will affect cocoapods, spm and carthage on ci)

currently 2 tests are failing on all sample apps

File: Cartfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
 
 github "marmelroy/PhoneNumberKit" ~> 3.3.0
-github "cko-payment-interfaces/checkout-ios" ~> 0.2.2
+git "./Checkout"
```
Comment (deepesh-vasthimal-cko) [inline Cartfile:3]: 👍

File: Checkout/Checkout.podspec [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Pod::Spec.new do |s|
+  s.name = 'Checkout'
+  s.version = '0.1.0'
+  s.summary = 'Checkout SDK for iOS'
+
+  s.description = <<-DESC
+TODO: Add long description of the pod here.
```
Comment (deepesh-vasthimal-cko) [inline Checkout/Checkout.podspec:7]: TODO Update

File: Checkout/Checkout.podspec [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                  DESC
+
+  s.homepage = 'https://github.com/checkout/checkout-sdk-ios'
+  s.license = { :type => 'MIT', :file => 'LICENSE' }
+  s.author = { "Checkout.com Integration" => "integration@checkout.com" }
+  s.source = { :git => 'https://github.com/checkout/checkout-sdk-ios.git', :tag => s.version }
```
Comment (deepesh-vasthimal-cko) [inline Checkout/Checkout.podspec:13]: Question : Are we still getting the checkout iOS SDK from different repo ?

**Review by harry-brown-cko — APPROVED** [review]

**Review by ehab-al-cko — APPROVED** [review]

**Pull request merged**